tests/System.Text.Utf8String.Tests/TestHelper.cs

```diff
@@ -22,7 +22,7 @@ public static void Validate(Utf8Span str1, Utf8Span str2)
             Assert.Equal(str1.Length, str2.Length);
             for (int i = 0; i < str1.Length; i++)
             {
-                Assert.Equal(str1[i], str2[i]);
+                Assert.Equal(str1.Bytes[i], str2.Bytes[i]);
```

Comment:
Can you use SequenceEqual instead of a for loop (element by element) comparison?